### PR TITLE
Fix aniso diffuse calculation in diffBragg

### PR DESCRIPTION
**Pull request by mewall**

### CODE REVIEW

File: simtbx/diffBragg/src/diffBragg_cpu_kernel.cpp [hunk 1]
```diff
@@ -189,19 +189,19 @@ void diffBragg_sum_over_steps(
...
                         for (int ll=0; ll <1; ll++){
                             Eigen::Vector3d H0_offset(h0+hh, k0+kk, l0+ll);
-                            Eigen::Vector3d Q0 = Ainv*H0_offset;
+                            Eigen::Vector3d Q0 = db_cryst.UMATS_RXYZ[mos_tic].transpose()*Ainv*H0_offset;
                             double exparg = 4*M_PI*M_PI*Q0.dot(db_cryst.anisoU*Q0);
                             //double dwf = exp(-exparg);
                             Eigen::Vector3d delta_H_offset = H_vec - H0_offset;
-                            Eigen::Vector3d delta_Q = Ainv*delta_H_offset;
+                            Eigen::Vector3d delta_Q = db_cryst.UMATS_RXYZ[mos_tic].transpose()*Ainv*delta_H_offset;
                             Eigen::Vector3d anisoG_q = db_cryst.anisoG*delta_Q;
```
Comment (Trzs) [inline simtbx/diffBragg/src/diffBragg_cpu_kernel.cpp:205]: calling global?

File: simtbx/diffBragg/src/diffBragg_gpu_kernel.cu [hunk 2]
```diff
@@ -372,19 +372,19 @@ void gpu_sum_over_steps(
                 MAT3 Amat = UBO;
                 MAT3 Ainv = UBO.inverse();
                 if (s_gamma_miller_units){
-                  anisoG = anisoG * Amat;
+                  anisoG = anisoG * Bmat_realspace;
```
Comment (Trzs) [inline simtbx/diffBragg/src/diffBragg_gpu_kernel.cu:375]: modifiying global matrix each call?

Comment (mewall) [reply simtbx/diffBragg/src/diffBragg_gpu_kernel.cu]: Moved this outside the threaded region and using a local copy. OK?

File: simtbx/diffBragg/src/diffBragg_cpu_kernel.cpp (hunk 1 above)
Comment (mewall) [reply simtbx/diffBragg/src/diffBragg_cpu_kernel.cpp]: Created a local copy. OK?

**Pull request merged**